src/NLog/Targets/ConsoleWordHighlightingRule.cs

```diff
@@ -116,7 +130,7 @@ public Regex CompiledRegex
                         }
                     }
 
-                    RegexOptions regexOptions = RegexOptions.Compiled;
+                    RegexOptions regexOptions = RegexOptions.None;
```

Comment:
This change does not yield a noticeable change in performance. The reason I changed this was due to a previous comment referring to the [MSDN documentation](https://msdn.microsoft.com/en-us/library/8zbs0h2f%28v=vs.110%29.aspx) stating that the `Compiled` option compiles the regular expression to MSIL code in an assembly which cannot be unloaded. I believe the concern was that this might cause increased memory usage in the long run. However, coming to think of it, if the regular expression does not change (which will typically be the case), this should not be an issue.

I personally do not think this makes much of a difference. I'll change it back if you want.